deployment/arm/deploy-global.sh

```diff
@@ -31,23 +44,26 @@ toArmArray() {
   printf $value
 }
 
-let counter=0
+let counter=0 || true
```

Comment:
why do we need the || true? can you add a comment to explain this

Reply:
After setting set -e to the script this expression evaluates to 0. Searched online and one way to solve this is to add || true.